contracts/TopShotLocking.cdc

```diff
@@ -0,0 +1,114 @@
+import NonFungibleToken from 0xNFTADDRESS
+
+pub contract TopShotLocking {
```

Comment:
Did you decide if you want this contract to be a general utility contract or not? If so, it should be something like TopShotUtilities or something like that

Reply:
I prefer to make it specific Locking for simplicity